ci/pbs/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "3"
```

Comment:
I need to use version: "2" with my version of docker-compose (1.8.0) (I followed the docker installation doc on a Ubuntu 16.04 box). Is there a reason you are using version: "3"?

Reply:
I don't think I need version 3 here, which is more for using docker stack deploy or things like that IIRC. I will update that.

I find it weird that you install such an old version of docker-compose though, following official doc should install 1.21 currently: https://docs.docker.com/compose/install/.

Reply:
Probably I only followed the install instructions for docker and thought that it would do docker-compose too ... I am guessing this is the docker-compose from `apt` on Ubuntu 16.04 ...